multipaz-zkp/src/androidTest/kotlin/org/multipaz/zkp/LongfellowNativesTest.kt

```diff
@@ -0,0 +1,13 @@
+package org.multipaz.zkp
+
+import org.multipaz.zkp.mdoc.longfellow.VerifierCodeEnum
+import kotlin.test.assertEquals
+import org.junit.Test
+
+class LongfellowNativesTest {
+    @Test
+    fun testFullVerificationFlow_success() {
+        val verifierCodeEnum = LongfellowNativeTestsCommon.runFullVerification()
+        assertEquals(VerifierCodeEnum.MDOC_VERIFIER_SUCCESS, verifierCodeEnum)
+    }
+}
```

Comment:
What's the difference between this and the similar test in jvmTest? Is it perhaps to make sure the test runs on both the device and the server?

Reply:
Yes exactly